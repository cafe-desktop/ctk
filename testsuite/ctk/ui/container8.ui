<!-- test invalid id in container markup -->
<interface>
  <object class="CtkBox">
    <child>
      <object class="CtkLabel" id="widget1"/>
      <packing>
        <property name="expand" translatable="yes" comments="foo" context="bar">True</property>
      </packing>
    </child>
    <focus-chain>
      <widget name="xyzzy"/>
    </focus-chain>
  </object>
</interface>
