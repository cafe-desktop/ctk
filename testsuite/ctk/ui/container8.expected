ERROR: ctk-builder-error-quark 13
.:11:29 Object with ID xyzzy not found
